backends/imgui_impl_wgpu.cpp

```diff
@@ -475,18 +478,19 @@ static void ImGui_ImplWGPU_CreateFontsTexture()
 
     // Upload texture data
     {
-        WGPUTextureCopyView dst_view = {};
+        WGPUImageCopyTexture dst_view = {};
         dst_view.texture = g_resources.FontTexture;
         dst_view.mipLevel = 0;
         dst_view.origin = { 0, 0, 0 };
-#if !defined(__EMSCRIPTEN__) || HAS_EMSCRIPTEN_VERSION(2, 0, 14)
         dst_view.aspect = WGPUTextureAspect_All;
-#endif
         WGPUTextureDataLayout layout = {};
         layout.offset = 0;
         layout.bytesPerRow = width * size_pp;
         layout.rowsPerImage = height;
         WGPUExtent3D size = { static_cast<uint32_t>(width), static_cast<uint32_t>(height), 1 };
+#if !defined(__EMSCRIPTEN__)
```

Comment:
ditto

Reply:
Not setting depth here leads to `wgpuQueueWriteTexture` not being executed properly (no texture data written), however, not error message is produced.